src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/ValidationMessages.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation.utils;
+
+public class ValidationMessages {
+
+    private ValidationMessages() {}
+
+    public static final String NOT_NULL_ERROR_MESSAGE = " must not be null";
```

Comment:
How about using %s in the error message content (like INVALID_EMAIL_ERROR_MESSAGE below) so as not to have to do the string concatenation, when used?

Reply:
Updated this, significant unit test changes needed.